documentation/Installing-Clear-Containers-on-Centos-7.md

```diff
@@ -0,0 +1,171 @@
+# Install the Clear Containers runtime on Centos 7
+
+
+## Introduction
+
+Installing Clear Containers on CentOS is slightly more difficult than on other distributions mainly due to the fact that it does not provide current versions of gcc and glibc.
+
+Hence, this guide explains how to install updated packages in /usr/local, to avoid conflicting with the base package
+
+## Warning
+
+Note carefully the comment in the section [Install container kernel and rootfs](#install-container-kernel-and-rootfs). You MUST update the packages listed regularly to ensure your Clear Containers benefit from performance improvements, new features and most importantly CVE security fixes.
+
+## Install updated glib and json-glib
+
+Note: these packages need to be built with the default CentOS compiler since they don't build by default with gcc 6.
+
+```
+$ export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
+$ sudo yum groupinstall 'Development Tools'
+$ sudo yum install zlib-devel libffi-devel
+
+$ curl -L -O "https://download.gnome.org/sources/glib/2.46/glib-2.46.2.tar.xz"
+$ tar -xvf glib-2.46.2.tar.xz
+$ pushd glib-2.46.2
+$ ./configure --prefix=/usr/local --disable-silent-rules && make -j5 && sudo make install
+$ popd
+
+$ curl -L -O "https://download.gnome.org/sources/json-glib/1.2/json-glib-1.2.2.tar.xz"
+$ tar -xvf json-glib-1.2.2.tar.xz
+$ pushd json-glib-1.2.2
+$ ./configure --prefix=/usr/local --disable-silent-rules && make -j5 && sudo make install
+$ popd
+
+```
+
+## Install updated gcc
+
+This is required for `qemu-lite`.
+
+```
+$ sudo yum install gmp-devel mpfr-devel libmpc-devel gcc-c++
+$ curl -L -O http://www.mirrorservice.org/sites/ftp.gnu.org/gnu/gcc/gcc-6.2.0/gcc-6.2.0.tar.gz
+$ tar xvf gcc-6.2.0.tar.gz
+$ pushd gcc-6.2.0
+$ ./configure --enable-languages=c --disable-multilib --prefix=/usr/local/gcc-6.2.0
+$ make -j5 && sudo make install
+$ popd
+```
+
+## Install qemu-lite
```

Comment:
To avoid possible confusion, it might be an idea to add a comment here stating that although this installs the qemu-lite version of qemu, the *binary* will still be called `qemu-system-x86_64`.